### ISSUE TITLE
Temperature scaling for Multi-label classification

### ISSUE THREAD

**Issue by aditya1709** [fabiankueppers/calibration-framework#11]
If we were to use Temperature scaling for Multi-label classification, do we work under the assumption that every class is independent of each other and perform calibration on each of our classes independently?

**Comment by fabiankueppers** [comment]
Hi,
in your scenario (temperature scaling, multi-label), a softmax is applied to the rescaled logits, thus the probabilities are treat as dependent. There is still a parameter called "independent_probabilities" but I'm afraid that this parameter has currently no effect on multi-class classification. Hope this helps.

**Comment by aditya1709** [comment]
@fabiankueppers Sorry for bringing this up again. 
Object Detection is essentially a multilabel problem (in terms of labels per frame perspective). So if I supplied same bounding box information for GT and pred but with actual confidence information for the class would 'Dependent Logistic Calibration' or 'Dependent Beta Calibration' methods work?
